### DISCUSSION TITLE
Standard Notes missing biometric step for web app

### DISCUSSION THREAD

**Discussion by dnbaxd92** [standardnotes/forum#2858] [General]
**Describe the bug**
Billion dollar insight, what's the point of Standard Notes or Apple Notes phone app biometric security layer e.g. Face ID, when cybercriminals can hack laptop and via website app to access and copy users' notes/data? Business opportunity to max security and profits for VIPs where each time user attempt to sign into website with email and password, and if user have biometric setting enabled, the website will prompt user to go to e.g. Standard Notes app on iPhone/iPhone Standard Notes pop up notification to trigger the biometric step for user to authenticate e.g. Face ID via phone. Once security passed via Phone, the Standard Notes iPhone app via internet tell its Standard Notes web app server to give user access to the data on user's laptop.

**To Reproduce**
NA 

**Expected behavior**
NA

**Screenshots**
NA

**Desktop (please complete the following information):**
Cross platforms

**Smartphone (please complete the following information):**
Cross platforms

**Additional context**
Legal, financial and privacy issues.

**Comment by cyb3rz3us** [comment]
> what's the point of Standard Notes or Apple Notes phone app biometric security layer...when cybercriminals can hack laptop and via website app to access and copy users' notes/data

This applies to any mobile app that also has a PC\Web component.  As such, each device and\or service incorporates their own set of security mechanisms and it is up to the user of each to use those mechanisms properly.

Said differently, if a SN user does a good job with the iPhone but a poor job with their web or PC password, that is a user issue; not a SN issue.

The 2 major schools of thought are **(a)** play the "parental" role and force the users into a given security paradigm; or **(b)** provide the users with options and let them decide what works best re: risk, ease of use, etc.  I am far more in favor of "B" in pretty much all cases except for things like nation-state, bank, etc. type opSec environments.

**Comment by dnbaxd92** [comment]
Eliminate the laptop environment and use powered tablet/phone with biometrics inbuilt features linked to computer monitor and external keyboard and mice to eliminate the risk?

**Comment by cyb3rz3us** [comment]
> Eliminate the laptop environment

So now you wants users to be forced to not use laptops?  Not really sure what you're trying to accomplish but that is horrible idea...
